src/ocamlorg_frontend/pages/playground.eml

```diff
@@ -0,0 +1,53 @@
+let render () =
+Layout.render
+~turbo_full_reload:true
```

Comment:
Makes me think: we should remove turbolink for now, until we have a better integration. Out of scope for this PR of course.